freqtrade/pairlist/AgeFilter.py

```diff
@@ -26,12 +26,11 @@ def __init__(self, exchange, pairlistmanager,
         self._min_days_listed = pairlistconfig.get('min_days_listed', 10)
 
         if self._min_days_listed < 1:
-            raise OperationalException("AgeFilter requires min_days_listed must be >= 1")
+            raise OperationalException("AgeFilter requires min_days_listed be >= 1")
```

Comment:
I think it should be `AgeFilter requires min_days_listed to be >= 1` (to be)

Reply:
No strong opinions on this, but I found the original wording to be fine.

Reply:
```suggestion
            raise OperationalException("AgeFilter requires min_days_listed to be >= 1")
```